docs/moderne-cli/README.md

```diff
@@ -12,30 +15,113 @@ Boot 2 to Spring Boot 3.
 
 ### Download and configure the Moderne CLI
 
-1. Go to the [Moderne platform](https://app.moderne.io) and sign in. If you don't have an account, you can sign up for
-   free.
+1. Go to the [Moderne platform](https://app.moderne.io) and sign in. If you don't have an account, you can sign up for free.
+
+2. Download the CLI for your operating system:
+   <Tabs
```

Comment:
I added these tabs in - but we don't have links yet for them. If we have those, please let me know and I can update them accordingly.

Reply:
Sure yes every release of the CLI uploads those to Azure:
https://github.com/moderneinc/moderne-cli/actions/runs/6264639030/job/17011761015#step:11:1

This is the latest for linux; there's a similar one for Mac; the one for Windows failed with a timeout & needs a rerun.
> https://pkgs.dev.azure.com/moderneinc/moderne_public/_packaging/staging/maven/v1/io/moderne/moderne-cli-linux/v0.5.5/moderne-cli-linux-v0.5.5